packaging/os/yum.py

```diff
@@ -596,15 +600,15 @@ def install(module, items, repoq, yum_basecmd, conf_file, en_repos, dis_repos):
                 module.fail_json(msg="Failure deleting temp directory %s, %s" % (tempdir, e))
             module.exit_json(changed=True)
 
-        changed = True
-
```

Comment:
This should remain and then set res['changed'] = changed at the bottom.  The reason is that yum could still have returned "Nothing to do" which will set changed = False.